### PR TITLE
Added the new API action

### PR DESCRIPTION
**Pull request by matheusbarbosasp**
Updated the API version for non hybrid machines.

### CODE REVIEW

**Comment by kwirkykat** [comment]
Why is this under Matheus? @ameyaiam3 Let me know if you're having permission issues?

File: source/templates/4-Action-Set-VMSS.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
                         },
 						{
                             "condition": "[equals(toLower(parameters('type')), toLower('Microsoft.Compute/virtualMachineScaleSets'))]",
-                            "apiVersion": "2018-11-20",
```
Comment (kwirkykat) [inline source/templates/4-Action-Set-VMSS.json:84]: I think the HybridCompute apiVersion can be updated as well. We should use the same apiVersion across all of them no?

File: source/templates/4-Action-Set.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
                     "resources": [
                         {
                             "condition": "[equals(toLower(parameters('type')), toLower('Microsoft.Compute/virtualMachines'))]",
-                            "apiVersion": "2018-11-20",
```
Comment (kwirkykat) [inline source/templates/4-Action-Set.json:50]: Same here. I think we should use the same updated apiVersion for HybridCompute as well

**Review by kwirkykat — CHANGES_REQUESTED** [review]

**Review by kwirkykat — APPROVED** [review]

**Comment by matheusbarbosasp** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Pull request merged**